presto-native-execution/presto_cpp/presto_protocol/WriteProtocol.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/connectors/hive/HiveWriteProtocol.h"
+
+namespace facebook::velox::connector {
+class CommitInfo;
+}
+namespace facebook::presto::protocol {
+
+class PrestoNoCommitWriteProtocol
```

Comment:
This class is specific to HiveConnector. Should it have Hive in the name?

Reply:
Revised